lnrpc/rpc.proto

```diff
@@ -461,6 +461,16 @@ message SendRequest {
     payment to the recipient.
     */
     string payment_request = 6;
+
+    // Cut off specifying a fee value above which path finding will be short circuited
+    oneof fee_cut_off {
+        // Abandon the route if fees greater than amt
+        bool absolute_cut_off = 7 [json_name = "absolute_cut_off"];
+        // Abandon the route if fees greater than limit_satoshis
+        uint64 limit_satoshis = 8 [json_name = "limit_satoshis"];
```

Comment:
Now that I know that "MilliSatoshi are the native unit of the Lightning Network", should this be changed to MilliSatoshis?